vllm/usage/usage_lib.py

```diff
@@ -0,0 +1,126 @@
+import os
+import torch
+import json
+import platform
+import pkg_resources
+import requests
+import datetime
+import psutil
+from threading import Thread
+from pathlib import Path
+from typing import Optional
+from enum import Enum
+
+_USAGE_STATS_FILE = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)),
+    'usage_stats.json')  #File path to store usage data locally
```

Comment:
Since the package can be installed in many different ways/places, it might make more sense to have one clear path such as in the user's directory for documentation. Maybe `~/.config/vllm/usage_stats.json` or `~/.cache/vllm/usage_stats.json`?

Reply:
Ah yes! @yhu422 please do `~/.config/vllm/usage_stats.json` and respect `XDG_CONFIG_HOME` env var.